src/features/pilot_management/PilotSheet/sections/hangar/components/MechListItem.vue

```diff
@@ -72,7 +72,7 @@
                     icon
                     class="fadeSelect"
                     :disabled="mech.Pilot.ActiveMech === mech"
-                    @click.stop="mech.Pilot.ActiveMech = mech"
+                    @click.stop="mech.Pilot.State.ActiveMech = mech"
```

Comment:
What is the purpose of this change? Doesn't Pilot.ActiveMech just call State.ActiveMech?
Are we just trying to save a overhead of another function call?

Reply:
Pilot.ActiveMech is a getter function, not a Setter.  This follows the ActiveMech setter pattern set elsewhere in the code.